backend/src/main/java/fr/anettehaferkorn/backend/model/WineQuery.java

```diff
@@ -0,0 +1,17 @@
+package fr.anettehaferkorn.backend.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class WineQuery {
+
+    private String occasion;
+    private String wineStyle;
+    private String region;
```

Comment:
I think you will want to use an enum here and only allow input that your filter will be able to handle well, most probably for all attributes of this model.